### PR TITLE
Adding new Flaky Test servicecomb-java-chassis

### PR DESCRIPTION
**Pull request by anantdahiya8**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -3253,6 +3253,10 @@ https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b2783
 https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b27837aa6fec,schemacrawler-utility,us.fatehi.utility.test.html.TableCellTest.td2,ID,InspiredAFix,https://github.com/schemacrawler/SchemaCrawler/pull/525,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,common,io.seata.common.util.CollectionUtilsTest.encodeMap,ID,Accepted,https://github.com/seata/seata/pull/1802,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,tm,io.seata.tm.api.transaction.TransactionInfoTest.testFieldGetSetFromJson,ID,Deleted,,https://github.com/seata/seata/commit/6c0c75939dedbf19c4a21a8ac7aa0fb98879a8fd
+https://github.com/apache/servicecomb-java-chassis,76813bc18a2513d6797794721d0bcf72dcc4ab22,foundations/foundation-vertx,org.apache.servicecomb.foundation.vertx.http.TestAbstractHttpServletRequest#testAttribute,ID,,,
```
Comment (darko-marinov) [inline pr-data.csv:3256]: The test name format is wrong, should use '.' (not '#') to separate test class name and method name.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
You should correct the format (and confirm that the checker passes).

**Review by darko-marinov — CHANGES_REQUESTED** [review]
The checker still fails; wrong sort order (see IDoFT README for more info on sorting).

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Can you please squash your two commits into one? If it's hard, let me know, and I can do it.

**Comment by anantdahiya8** [comment]
> Can you please squash your two commits into one? If it's hard, let me know, and I can do it.

Done

**Pull request merged**